assets/models/system/CodeLlama-7b-Python-hf/spec.yaml

```diff
@@ -5,18 +5,26 @@ properties:
   SHA: b1d96d612f1dcaf97ee3cc98c4975f6169909d6a
   inference-recommended-sku: Standard_NC12s_v3, Standard_NC24s_v3, Standard_NC24rs_v3,
     Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
+  evaluation-recommended-sku: Standard_NC12s_v3
   languages: EN
 tags:
   Featured: ''
   Preview: ''
+  computes_allow_list:
```

Comment:
I think all inference allow list SKUs can be added here.

Reply:
removed this tag